const exceptions = {
    "auth/email-already-exists": "The email address is already in use.",
};
